src/App.js

```diff
@@ -1,35 +1,21 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import './index.scss';
 import MovieContainer from './movie-components/movie-container-component/movie-container.js';
 import Header from './header-components/header-component.js';
+import SingleMovieView from './single-movie-component/single-movie-component.js';
 import fetchRequests from './fetch-requests.js';
 
-
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      singleMovie: [],
-      filteredMovies: [],
-      beingSearched: false,
       movies: [],
       singleMovieView: false,
       isLoading: true,
+      singleMovie: []
```

Comment:
Interesting that this is an array, rather than an object - is there a reason for this?